ocrd/ocrd/cli/workspace.py

```diff
@@ -10,7 +10,7 @@
 import click
 
 from ocrd import Resolver, Workspace, WorkspaceValidator, WorkspaceBackupManager
-from ocrd_utils import getLogger, initLogging, pushd_popd, EXT_TO_MIME
+from ocrd_utils import getLogger, initLogging, pushd_popd, EXT_TO_MIME, resolve_mets_arguments
```

Comment:
Why not in `ocrd.resolver` or `ocrd.workspace`?

Reply:
Mostly because functions are easier to test than methods. I was thinking of making it part of the `Resolver` since that has to be instantiated where the method is used. Would that be better?

Reply:
Yes, that would look better IMHO.

Reply:
56edc4c9 makes it a method of resolver and 187bc7fc makes sure the instantiation of the workspace uses the results of `Resolver.resolve_mets_arguments` as-is, without doing any `abspath` or `basename` calls.